src/firefly/js/ui/AppPropertiesCtx.jsx

```diff
@@ -6,6 +6,5 @@ export const AppPropertiesCtx = React.createContext( {
     landingPage: undefined,
     footer: undefined,
     showUserInfo: false,
-    showLandingHelp: true,
```

Comment:
Removed it because now same can be achieved by `<LandingPage slots={{tabsMenuHint: false or null}}/>`